ext/bg/js/search-query-parser.js

```diff
@@ -86,6 +86,7 @@ class QueryParser extends TextScanner {
 
     setOptions(options) {
         super.setOptions(options);
+        super.setEnabled(true);
```

Comment:
I actually allowed query parser to be disabled on purpose when scanning is disabled, but it would probably make more sense if the option only applied to website content because the search page still works normally. The option description says "Enable content scanning" which is a bit ambiguous.

Reply:
To me, it feels like there should be a distinction between the two. It's odd that the search page will still work and display the full text from the input, but you won't be able to scan it. Since the page is dedicated to do this task, IMO it would be fine to leave it always on. I can revert if you like, but that was my line of thinking.

Reply:
I agree with you, I think it's ok to leave the exception there. It didn't feel like a big deal when I did it and I even thought it could be desirable sometimes to also disable scanning here, but it could also be very confusing without having any indication that it's disabled. This is a much better solution than implementing some kind of indicator for the unverified use case.